tooling/azure-automation/resources-cleanup/src/resources_cleanup.py

```diff
@@ -52,6 +52,20 @@ def time_delta_greater_than_two_days(now: datetime.datetime, resource_group_crea
 
     return n_days > 2
 
+def time_delta_greater_than_one_month(now: datetime.datetime, resource_group_creation_time: datetime.datetime):
+    if now is None:
```

Comment:
This can not happen with the types used. If it could happen use `Optional` type in function parameter

Reply:
Refactored in https://github.com/Azure/ARO-HCP/pull/3078/commits/87a20307357ddcf8d49a131de5968922257faa78